utils/tx.ts

```diff
@@ -444,6 +444,17 @@ export const createPool = async (
   return result;
 };
 
+export const promotePool = async (sudoAccount: KeyringPair, liqAssetId: BN) => {
+  testLog.getLog().info(`Promoting pool :${liqAssetId}`);
+  const mangata = await getMangataInstance();
+  const api = await mangata.getApi();
+  const result = await signSendAndWaitToFinishTx(
```

Comment:
careful! 
Sudo operations must be handled with nounce Db!

Reply:
like this:
https://github.com/mangata-finance/mangata-e2e/pull/154/files#diff-c7b01ba73caeee9e0c424b9cbc65030d22db0d866a0a4254bb1f60db624c06b5R975